themes/envy/envy.theme.bash

```diff
@@ -6,11 +6,24 @@ SCM_THEME_PROMPT_SUFFIX="${green}|"
 
 GIT_THEME_PROMPT_DIRTY=" ${red}✗"
 GIT_THEME_PROMPT_CLEAN=" ${bold_green}✓"
-GIT_THEME_PROMPT_PREFIX=" ${green}|"
+GIT_THEME_PROMPT_PREFIX=" ${green}| "
 GIT_THEME_PROMPT_SUFFIX="${green}|"
 
+# virtualenv prompts
+VIRTUALENV_CHAR="ⓔ "
+VIRTUALENV_THEME_PROMPT_PREFIX=""
+VIRTUALENV_THEME_PROMPT_SUFFIX=""
+
+# Display virtual environment info
+function virtualenv_prompt {
+  if [[ -n "$VIRTUAL_ENV" ]]; then
+    virtualenv=`basename "$VIRTUAL_ENV"`
+    echo -e "$VIRTUALENV_CHAR$virtualenv "
+  fi
+}
+
 function prompt_command() {
-    PS1="\n${yellow}$(ruby_version_prompt) ${purple}\h ${reset_color}in ${green}\w\n${bold_cyan}$(scm_char)${green}$(scm_prompt_info) ${green}→${reset_color} "
+    PS1="\n${yellow}$(virtualenv_prompt)${purple}\h ${reset_color}in ${green}\w\n${bold_cyan}$(scm_char)${green}$(scm_prompt_info) ${green}→${reset_color} "
```

Comment:
You probably shouldn't remove the `ruby_version_prompt` as other users  might rely on it - If you're not using it, it should just quietly hide - Are you seeing some other issue?

I would probably remove the trailing space after the ruby prompt though, leaving us with something like:
```
    PS1="\n${yellow}$(virtualenv_prompt){yellow}$(ruby_version_prompt)${purple}\h ${reset_color}in ${green}\w\n${bold_cyan}$(scm_char)${green}$(scm_prompt_info) ${green}→${reset_color} "
```

Again please say if you are sing some other kind of issue with the ruby prompt.

Reply:
Thx,  the ruby_version_prompt disable prompt is very nice option for me, and I update this PR for discuss.

Reply:
Usage is at .bash_profile config file, export RUBY_VERSION_PROMPT_DISABLED="1" can hide ruby version prompt.